src/window/mod.rs

```diff
@@ -235,9 +237,12 @@ pub fn main_loop(
     event_loop: EventLoop<UserEvent>,
 ) -> Result<(), EventLoopError> {
     let cmd_line_settings = SETTINGS.get::<CmdLineSettings>();
+    let render_thread_done = Arc::<AtomicBool>::new(AtomicBool::new(false));
```

Comment:
AtomicBool isn't necessary because the type can be inferred by the constructor argument
```suggestion
    let render_thread_done = Arc::new(AtomicBool::new(false));
```

Reply:
Good point, I will simplify that!

Reply:
Simplified!